new/client-rb-logger.md

```diff
@@ -0,0 +1,39 @@
+---
+RFC: unassigned
+Title: Logging for chient.rb
+Author: Zeal Jagannatha <zeal@fb.com>
+Status: Draft
+Type: Standards Track
+Tracking:
+  - https://github.com/chef/chef/issues/5526
+---
+
+# Logging for client.rb
+
+Output from client.rb doesn't get logged to the log file specified with --logfile.
+If you want to save the output from client.rb along-side the chef logs, you need to redirect stdout, rather than using chef's built-in logging.
+I'd like to make Chef::Log accessible from the client.rb so messages logged during client.rb execution can be stored using chef's built-in logging.
+
+## Motivation
+
+    As a chef user,
```

Comment:
Nope. We want to log shit. We want to log which handlers we loaded. We want to log whether your in taste-tester or prod chef (and taste-tester is used various places outside of Facebook).

Today, we `puts "INFO: ..."` bla bla. and we have a wrapper to chef which we're looking to opensource which just runs `chef-client $options > /var/log/chef/chef.$DATETIMESTAMP` (among several other things).

I was discussing this with @lamont-granquist  at Summit and we agreed that about 90% of the things the wrapper does can/should be in core-chef. We started looking at the logging aspect first, and realized that we would lose any logging we make from client.rb which would be a bummer.